src/test/java/org/jenkinsci/plugins/workflow/graph/FlowNodeTest.java

```diff
@@ -121,4 +127,303 @@ private static void assertActiveSteps(WorkflowRun b, String... expected) {
         assertEquals(Arrays.asList(expected), actual);
```

Comment:
You were able to leave my existing test untouched—that is encouraging. :-)

Reply:
@jglick As they say around here "if it ain't broke don't fix it."